tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -1148,6 +1148,8 @@ handle_build() {
 		echo "Pushing ${build_target} with tags: ${tags[*]}"
 
 		for tag in "${tags[@]}"; do
+			# tags can only contain lowercase and uppercase letters, digits, underscores, periods, and hyphens
+			tag=("$(echo ${tag} | tr -c '[a-zA-Z0-9\_\.\-]' _)")
```

Comment:
Did it come from a real problem? or are you just making it fail-proof?

Reply:
Real problem - the pause image is `docker://registry.k8s.io/pause-3.9` at the moment, which isn't valid, but I figured it might occur for versions tags & branch names that are less restrictive than container image tags.